languages.toml

```
@@ -1448,3 +1448,17 @@ indent = { tab-width = 2, unit = "\t" }
 [[grammar]]
 name = "openscad"
 source = { git = "https://github.com/bollian/tree-sitter-openscad", rev = "5c3ce93df0ac1da7197cf6ae125aade26d6b8972" }
+
+[[language]]
+name = "prisma"
+scope = "source.prisma"
+injection-regex = "prisma"
+file-types = ["prisma"]
+roots = ["package.json"]
+comment-token = "//"
+language-server = { command = "prisma-language-server", args = ["--stdio"] }
+indent = { tab-width = 2, unit = "  " }
+
+[[grammar]]
+name = "prisma"
+source = { git = "https://github.com/victorhqc/tree-sitter-prisma", rev = "0ac307e4be0409d4327082ddc737ed432a49dfc3" }
```

```suggestion
source = { git = "https://github.com/victorhqc/tree-sitter-prisma", rev = "a3eaf13e75683cb22269814bf5e87af7eb671b1c" }
```

That issue got fixed so with this I think we'll be good to go 🙂

---

I'm not sure that got fixed

---

Can you check again with this commit? `36a8e66164babe9f588489c4d9f2b11ddb520ff3` If it stops happening, then I'll merge [this PR](https://github.com/victorhqc/tree-sitter-prisma/pull/39) and build a new release.

---

Released [v1.3.0](https://github.com/victorhqc/tree-sitter-prisma/releases/tag/v1.3.0) should not have the seg fault anymore.